packages/hub-score/README.md

```diff
@@ -0,0 +1,70 @@
+
+
+## Description
+
+This project aims to extract the credit score of each wallet address entered,and ensure that it qualifies for a loan.
+
+
+## Installation
+
+```bash
+$ yarn install
+```
+
+# Setting up environment variables
+
+you need create a env in the score-cripto-main project in packages
```

Comment:
this is deprecated

Reply:
i changed for hub-score 
![image](https://github.com/Quantum3-Labs/Stormbit-monorepo/assets/126603997/94ae2d77-e9af-4c3b-9228-6402014a7102)